apps/src/storage/levelbuilder/ManifestEditor.jsx

```diff
@@ -9,9 +27,41 @@ class ManifestEditor extends React.Component {
   };
 
   render() {
+    const isValidJson =
+      this.props.libraryManifest.categories &&
+      this.props.libraryManifest.tables;
+
+    const showUnpublishedTables = experiments.isEnabled(
+      experiments.SHOW_UNPUBLISHED_FIREBASE_TABLES
+    );
+    const categories = (this.props.libraryManifest.categories || []).filter(
+      category => showUnpublishedTables || category.published
+    );
     return (
       <div>
         <h1>Edit Dataset Manifest </h1>
+        <h2>Library Preview</h2>
+        {showUnpublishedTables && (
+          <p style={styles.warning}>
+            Note: Showing unpublished categories and tables. To hide, turn off
+            the experiment by adding
+            ?disableExperiments=showUnpublishedFirebaseTables to the URL.
```

Comment:
@Hamms - now that we're working on platformization, is there any plan to translate levelbuilder?

Reply:
I was wondering about this too. My gut was that since we won't be translating LB at least for a while, it probably makes sense to keep these out of the translation pipeline for now, and we can always add them later.

Reply:
There certainly will be at some point, but I don't think we've started scheduling that work yet; certainly at this point you don't need to worry about it